content/en/docs/Getting Started/_index.md

```diff
@@ -393,6 +393,7 @@ Now you have the core component of Tekton, Tekton Pipelines, installed on
 your Kubernetes or OpenShift cluster with the Tekton CLI installed on your local
 machine. If you would like to install more components, see the list below:
 
+* [Getting Started - Pipelines](./pipelines)
```

Comment:
It doesn't fill like this link belongs to this list.

You could say:

"Continue to experiment with pipelines with the  [Getting Started - Pipelines](./pipelines).
If you would like to install more components, see the list below:"

Reply:
fixed.